.gitignore

```diff
@@ -14,7 +14,6 @@
 x64/
 build/
 [Oo]bj/
-bin/

```

Comment:
Doesn't that result in committing all the built binaries to the repo?

Reply:
I'll revert that one